src/sentry/models/group.py

```diff
@@ -825,6 +825,15 @@ def message_short(self):
     def organization(self):
         return self.project.organization
 
+    @property
+    def sdk(self) -> str | None:
```

Comment:
For the sake of simplicity I'm calling it just `sdk` not `sdk_name` nor `sdk_name_normalized`.